src/main/java/com/cloudbees/jenkins/plugins/kubernetes_credentials_provider/SecretUtils.java

```diff
@@ -122,6 +132,23 @@ public static CredentialsScope getCredentialScope(Secret s) throws CredentialsCo
         return scope;
     }
 
+    /**
+     * Gets all item-group scopes from a secret.
+     * Format of the annotation: ["/job/thisIsJobA/", "/job/thisIsJobB/"]
+     *
+     * @param s the secret whose item-group scope we want to obtain.
+     * @return a set of all item-group scopes
+     */
+    public static Set<String> getCredentialItemScopes(Secret s) {
+        String itemListString = s.getMetadata().getAnnotations().get(SecretUtils.JENKINS_IO_CREDENTIALS_ITEM_GROUP_ANNOTATION);
+        if (itemListString == null) {
+            return Collections.emptySet();
+        }
+        itemListString = itemListString.trim();
+        itemListString = itemListString.substring(1, itemListString.length()-1);
+        return Sets.newHashSet(Splitter.on(", ").trimResults().omitEmptyStrings().split(itemListString));
```

Comment:
please avoid the use of guava, its usage is generally discouraged in Jenkins plugins.